tests/queries/0_stateless/03165_database_and_table_count_limitation_sql

```diff
@@ -0,0 +1,21 @@
+create database db1;
```

Comment:
Don't do this .... we only have `.sql` and `.reference` in `0_stateless` directory